docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,442 @@
+
+<!---  Tabs start here -->
+
+<Tabs
+  defaultValue="BAM-paired"
+  values={[
+    { label: 'BAM (paired)', value: 'BAM-paired', },
+    { label: 'BAM (single)', value: 'BAM-single', },
+    { label: 'FASTQ (paired)', value: 'FASTQ-paired', },
+    { label: 'FASTQ (single)', value: 'FASTQ-single', },
+  ]
+}>
+<TabItem value="BAM-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+  	"read_group_id_in_bam": "C0HVY.2",
+  	"submitter_read_group_id": "C0HVY.2",
+  	"platform_unit": "74_8a",
+  	"is_paired_end": true,
+  	"file_r1": "test_rg3.bam",
+  	"file_r2": "test_rg4.bam",
+  	"read_length_r1": 150,
+  	"read_length_r2": 16,
+  	"insert_size": 232,
+  	"sample_barcode": null,
+  	"library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="BAM-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": "C0HVY.2", 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8a",
+    "is_paired_end": true,
+    "file_r1": "test_rg55.bam",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null,
+    "insert_size": 232,
+    "sample_barcode": null,
+    "library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": true,
+    "file_r1": "test_rg23.fq",
+    "file_r2": "test_rg24.fq",
+    "read_length_r1": 150,
+    "read_length_r2": 125, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": false,
+    "file_r1": "test_rg5.fq",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+### Understanding the Payload Fields
+:::note   
+![Required](/assets/submission/dictionary-required.svg) indicates a required field that **must** be included in the payload or it will immediately fail submission.
+:::
+
+The metadata payload is broken down into 4 sections: `experiment`, `samples`, `read groups` and `files`, with all other fields being at the root level. Each section **must** be submitted in the payload.
+
+
+|  Payload Field     | Payload Section |  Attribute                                                |  Description                                                                                                                                                                  |  Permissible Values     |  |
```

Comment:
remove blank column..or add examples in there for things that require a specific format, like date

Reply:
blank column is removed